ghost/admin/app/controllers/explore.js

```diff
@@ -1,13 +1,23 @@
 import Controller from '@ember/controller';
 import {action} from '@ember/object';
+import {isEmpty} from '@ember/utils';
 import {inject as service} from '@ember/service';
+import {task} from 'ember-concurrency';
 
 export default class ExploreController extends Controller {
     @service explore;
     @service router;
+    @service store;
+
+    integrations = null;
```

Comment:
This can be set directly to `this.store.peekAll('integration')`, makes it easier to reason about because you don't then have to find where it's been set.